docs/02-app/02-api-reference/01-components/image.mdx

```diff
@@ -152,7 +152,7 @@ export default function Page() {
 
 <AppOnly>
 
-> **Good to know**: Using props like `loader`, which accept a function, require using [Client Components](/docs/app/building-your-application/rendering/client-components) to serialize the provided function.
+> **Good to know**: Using props like `loader`, which accepts a function, requires using [Client Components](/docs/app/building-your-application/rendering/client-components) to serialize the provided function.
```

Comment:
I don't think this is right. If you remove `like loader`, then the grammar was correct

Reply:
@styfle 

oh, I understand. I've removed 's' in `require`.